radixdlt-java/src/main/java/com/radixdlt/client/application/translate/ActionExecutionException.java

```diff
@@ -13,16 +13,30 @@
 	private final JsonObject errorData;
 
 	private ActionExecutionException(JsonObject errorData, List<ActionExecutionExceptionReason> reasons) {
-		super(reasons.toString());
+		super(reasons != null && !reasons.isEmpty()
+		? reasons.toString()
```

Comment:
Shouldn't this be intended?

Reply:
Ah! These spaces vs. tabs are killing me :)